index.js

```diff
@@ -3,7 +3,6 @@ const aws = require('aws-sdk')
 const fs = require('fs')
 
 const outputPath = core.getInput('OUTPUT_PATH')
-const secretName = core.getInput('SECRET_NAME')
```

Comment:
I think `secretName` was removed make this action failed. How about recover it?

Reply:
Looks like that. We will have a look at it